server/src/server.js

```diff
@@ -186,6 +196,132 @@ app.get("/:id/:domain", function (req, res) {
   });
 });
 
+const oAuthBaseURI = 'http://127.0.0.1:9010/v1',
+  contentBaseURI = 'http://127.0.0.1:3030',
+  profileBaseURI = 'http://127.0.0.1:1111/v1',
+  oAuthClientId = 'ac7ee3c317531aab',
+  oAuthClientSecret = 'eb7c92bd4616a7c1f86595492f360f55cc453974b062e71456d2d4b104f307f8';
```

Comment:
I guess these might cause problems being in the source?

Generally I think we should just read them from process.env.  We can make sure they are set properly on the live server.  I'm not sure what to do in development?